packages/data/src/onboarding/selectors.ts

```diff
@@ -112,3 +122,78 @@ export type ProfileItemsState = {
 	theme: string | null;
 	wccom_connected: boolean | null;
 };
+
+export type FieldLocale = {
+	locale: string;
+	label: string;
+}
+
+export type MethodFields = {
+	name: string;
+	option?: string;
+	label?: string;
+	locales?: FieldLocale[];
+	type?: string,
+	value?: string;
+}
+
+export type PaymentMethodsState = {
+	locale: string;
+	title: string;
+	content: string;
+	key: string;
+	image: string;
+	is_visible: boolean|RuleProcessor[];
+	plugins: string[];
+	is_configured: boolean|RuleProcessor[];
+	fields: MethodFields[];
+	api_details_url: string;
+	manage_url: string;
+};
+
+// TODO move out the following types
```

Comment:
These aren't meant to be here. I'm thinking that it should live somewhere that's shared, but I'm not sure where yet.